src/agoradatatools/etl/transform/biomarkers.py

```diff
@@ -0,0 +1,73 @@
+"""
+This module contains the transformation logic for the biomarkers dataset.
+This is for the Model AD project.
+"""
+
+import pandas as pd
+
+
+def transform_biomarkers(datasets: dict) -> list:
+    """
+    Takes dictionary of dataset DataFrames, extracts the biomarkers
+    DataFrame, and transforms it into a list of dictionaries grouped by
+    'model', 'type', 'ageDeath', 'tissue', and 'units'.
+
+    Args:
+        datasets (dict[str, pd.DataFrame]): dictionary of dataset names mapped to their DataFrame
+
+    Returns:
+        list[dict[str, Any]]: a list of dictionaries containing biomarker data modeled after intended final JSON structure
+    """
+    biomarkers_dataset = datasets["biomarkers"]
+
+    # Check that the dataset looks like what we expect
+    if not isinstance(biomarkers_dataset, pd.DataFrame):
```

Comment:
See my comment on your test function, this error check probably isn't necessary.

Reply:
Yes, I was thinking about this earlier. The type hints should catch this :)

I'll remove it.